yang/frr-nexthop.yang

```diff
@@ -27,10 +32,19 @@ module frr-nexthop {
       "Initial revision.";
   }
 
-  typedef gateway-address {
-    type inet:ip-address;
+  typedef optional-ip-address {
+     type union {
+       type inet:ip-address;
+       type string {
+         pattern
+           ' ';
```

Comment:
Shouldn't this be completely empty?

Reply:
were experimenting with pattern and kept one space. We can keep it empty and will update it.

Reply:
@qlyoung done I have updated it.